data-structures-linked-lists/get-tail.js

```diff
@@ -0,0 +1,11 @@
+export default function getTail(list) {
+  if (!list) {
+    return null;
+  }
+  let currentNode = list;
```

Comment:
This is safe, but you could just use `list` since you don't reference it anywhere.